.eslintrc.js

```diff
@@ -16,10 +17,7 @@ module.exports = {
     '@typescript-eslint/restrict-plus-operands': 'off',
     '@typescript-eslint/strict-boolean-expressions': 'off',
     'comma-dangle': 'off',
-    'dot-notation': 'off',
-    'import/first': 'off',
```

Comment:
Why was this removed❓

Reply:
@paldepind notice that most rules listed here are actually turning off rules that are turned on in the shareable config we extend from. So removing them is actually adding them.